src/main/java/org/checkerframework/checker/dependencyinjection/qual/Bind.java

```diff
@@ -8,22 +8,22 @@
 import org.checkerframework.framework.qual.SubtypeOf;
 
 /**
- * If an expression has type {@code @Bind({"foo"})}, then {@code foo.class} has been passed as an
- * argument to {@code bind} in a Guice module.
+ * Represents classes and names that have been passed as arguments to {@link
+ * com.google.inject.Binder#bind()}.
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.TYPE_USE, ElementType.TYPE_PARAMETER})
 @SubtypeOf({})
 @DefaultQualifierInHierarchy
 public @interface Bind {
+  /** The fully-qualified name of this annotation. */
+  static final String NAME = "org.checkerframework.checker.dependencyinjection.qual.Bind";
   /**
-   * Classes that have definitely been passed as an arugment to a {@code bind} call whose result is
-   * the expression whose type is annotated. The arguments are "fully qualified binary names"
-   * ({@link org.checkerframework.checker.signature.qual.FqBinaryName}): a primitive or <a
-   * href="https://docs.oracle.com/javase/specs/jls/se17/html/jls-13.html#jls-13.1">binary name</a>,
-   * possibly followed by some number of array brackets.
+   * Class names that have definitely been passed as an argument to a {@link
+   * com.google.inject.Binder#bind()} call.
```

Comment:
Why did you remove the note about the format of the argument? I think it is important to document it.

Also, this documentation should note that you expect that this will only ever have zero or one elements in the array, but that you used an array for compatibility with `AccumulationAnnotatedTypeFactory`.

Reply:
It definitely is important. I didn't know what to keep and what to leave out. I'll add the format and array size back into it.